lib/schemas/store-order-recipient.js

```diff
@@ -0,0 +1,14 @@
+const yup = require('yup');
+const config = require('../config');
+
+const storeOrderRecipientSchema = yup.object().shape({
+  id: yup.number().required(),
+  accept_language: yup
+    .string()
+    .oneOf(config.get('supportedLocales'))
+    .required(),
+});
```

Comment:
Given these are required, I think the method's documentation should reflect that

Reply:
done